### ISSUE TITLE
auto_edging_duration_minutes should not be limited to 300 mins.

### ISSUE THREAD

**Issue by LanceLake** [MausTec/edge-o-matic-3000#93]
For those who like the idea of having it go all night, 5 hours isn't going to cut it. Allow us to set 720 at least if there needs to be a max limit. 9999 would be preferable, but at least 720 is 12 hours, so it needs to be "reset" twice a day for those who want to do this multi-days.

**Comment by B4ben-69** [comment]
see pull request #92

**Comment by LanceLake** [comment]
Yes. This request still applies since that pull isn't approved yet.

**Comment by MauAbata** [comment]
Until the UI can be updated, manually adjust config.json with the correct value.
